ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+Browser:
```

Comment:
I say yes. If we don't need it fine, but I'd rather set the expectation that we _do_ need it. Then we don't need to ask later. Just makes things faster.